modyn/config/schema/pipeline/trigger/common/batched.py

```diff
@@ -10,3 +11,18 @@ class BatchedTriggerConfig(ModynBaseModel):
             "estimation, accuracy evaluation, drift detection, ...) should be performed."
         )
     )
+    warmup_intervals: int | None = Field(
+        None,
+        description=(
+            "The number of intervals before starting to use the main policy. Some "
+            "`DecisionCriteria` use this to calibrate the threshold. During the warmup, a simpler `warmup_policy` "
```

Comment:
DecisionCriteria come from drifts, right? Should we change this since this docstirng is more general now?

Reply:
Performance Triggers also have them. Cost triggers don't so I'll adjust that